doc/source/WorkingWithThePythonScriptingAPI/TutorialNodeGraphEditingInPython/index.md

```diff
@@ -157,15 +157,15 @@ import imath
 myShader['parameters']['Cs'].setValue( imath.Color3f( 0.25, 0.75, 0.25 ) )
 ```
 
-![](images/nodeEditorOpenGLPlug.png "The OpenGL node with an adjusted constant plug")
+![](images/nodeEditorOpenGLPlug.png "The OpenGL node with an adjusted Constant plug")
```

Comment:
I think it should be `Cs` plug if anything, as `constant` here is just saying "the plug that controls the constant color of the result thats actually called something else".